src/otx/algo/anomaly/openvino_model.py

```diff
@@ -20,10 +20,9 @@
 from lightning import Callback, Trainer
 from torchvision.transforms.functional import resize
 
-from otx.core.data.entity.anomaly import AnomalyClassificationDataBatch
+from otx.data.torch import TorchDataItemBatch
```

Comment:
can we think about alternative names for this? 
`TorchDataBatch`, `TorchBatch` or maybe just `Batch`? Thoughts?

Reply:
Yeah `TorchDataItemBatch` might be too verbose

Reply:
I prefer TorchDataItem and TorchDataBatch